ocaml/xapi/xapi_globs.ml

```diff
@@ -1364,8 +1366,16 @@ let other_options =
     , (fun () -> string_of_int !evacuation_batch_size)
     , "The number of VMs evacauted from a host in parallel."
     )
+  ; ( "ignore-vtpm-unimplemented"
+    , Arg.Set ignore_vtpm_unimplemented
+    , (fun () -> string_of_bool !ignore_vtpm_unimplemented)
+    , "Do not raise errors on use-cases where VTPM codepaths are not finished."
+    )
   ]
 
+(* The options can be set with the variable xapiflags in /etc/sysconfig/xapi.
```

Comment:
I think they can also be set in `/etc/xapi.conf`, that is what I usually change.